app/shared/github_handler.rb

```diff
@@ -34,8 +34,22 @@ module FastlaneCI
   # TODO: handle if the user doesn't have write permission and they are trying to do some writing
   module GitHubHandler
     include FastlaneCI::Logging
-    def github_action(&block)
+    def github_action(client, &block)
       retry_count ||= 0
```

Comment:
Could we use `retry_count = 0` please?

Reply:
Afaik when Rey retry is called the variable context is preserved, so it would be zero over and over.